src/routes/RequestsRoute.js

```diff
@@ -1130,6 +1240,27 @@ class RequestsRoute extends React.Component {
   }
 
   render() {
+    const getColumnMapping = {
+      select: <Checkbox
+        checked={this.getIsAllRowsSelected()}
+        aria-label={<FormattedMessage id="ui-requests.instances.rows.select" />}
+        onChange={this.toggleAllRows}
+      />,
+      requestDate: <FormattedMessage id="ui-requests.requests.requestDate" />,
+      title: <FormattedMessage id="ui-requests.requests.title" />,
+      year: <FormattedMessage id="ui-requests.requests.year" />,
+      itemBarcode: <FormattedMessage id="ui-requests.requests.itemBarcode" />,
+      callNumber: <FormattedMessage id="ui-requests.requests.callNumber" />,
+      type: <FormattedMessage id="ui-requests.requests.type" />,
+      requestStatus: <FormattedMessage id="ui-requests.requests.status" />,
+      position: <FormattedMessage id="ui-requests.requests.queuePosition" />,
+      servicePoint: <FormattedMessage id="ui-requests.requests.servicePoint" />,
+      requester: <FormattedMessage id="ui-requests.requests.requester" />,
+      requesterBarcode: <FormattedMessage id="ui-requests.requests.requesterBarcode" />,
+      singlePrint: <FormattedMessage id="ui-requests.requests.singlePrint" />,
+      proxy: <FormattedMessage id="ui-requests.requests.proxy" />
+    };
```

Comment:
It should be removed. There is a reason why we have a code duplication.
You just need to change  `getColumnMapping` to constant and rename it to something like 'columnLabels' or something similar.